devices/Hdc1080/Hdc1080.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Hdc1080</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Hdc1080</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework,dotnetfoundation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <readme>docs\README.md</readme>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes the .NET IoT Core binding Iot.Device.Hdc1080 for .NET nanoFramework C# projects.</description>
+    <tags>nanoFramework C# csharp netmf netnf Iot.Device.Hdc1080</tags>
+    <dependencies>
+      <dependency id="nanoFramework.CoreLibrary" version="1.12.0-preview.1" />
+      <dependency id="nanoFramework.System.Device.I2c" version="1.0.3-preview.4" />
+      <dependency id="nanoFramework.System.Math" version="1.4.4-preview.5" />
+      <dependency id="UnitsNet.nanoFramework.Length" version="4.115.0" />
```

Comment:
please check the nuspec here, it should match what you have in the nfproj and package.json. The build will let you know if it's broken anyway. But you'll have to adjust this manually. (you don't need the git versionning here)

Reply:
Fixed